vector/src/main/java/im/vector/app/features/analytics/VectorAnalytics.kt

```diff
@@ -70,4 +71,9 @@ interface VectorAnalytics {
      * Track a displayed screen
      */
     fun screen(screen: VectorAnalyticsScreen)
+
+    /**
+     * Update user specific properties
+     */
+    fun updateUserProperties(identity: Identity)
```

Comment:
@bmarty I'll also move this to the `AnalyticsTracker`

Reply:
https://github.com/vector-im/element-android/pull/5009/commits/928354312a69662963889a08f53eff893304a524